extensions/amp-story/1.0/amp-story.css

```diff
@@ -433,6 +433,14 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
                        "lower-third" !important;
 }
 
+amp-story-grid-layer.i-amphtml-has-page-attachment.i-amphtml-has-quiz {
+  padding-bottom: 104px !important;
```

Comment:
Nit: This (and below) should be overridable

Reply:
Removed the `!important` rule and added it to the overridable file